_posts/2014-11-13-cockpit-plugin-tutorial.md

```diff
@@ -78,92 +86,105 @@ Lets take a look at the pinger HTML, and see how it works.
     <title>Pinger</title>
```

Comment:
GitHub won't let me make a suggestion for a collapsed block. Bummer.

Anyway, the HTML is missing an doctype and `<html>` tag at the beginning.

Please add the following to the top of the HTML block:

```
<!doctype html>
<html lang="en">
```

(Also please check to make sure this is in the actual file that's downloaded too.)

Reply:
Indeed, I sent https://github.com/cockpit-project/cockpit/pull/14029 to fix that. I'll already include it in my follow-up.